ui: make two level footer

ppanero opened the pull request:
Closes https://github.com/inveniosoftware/invenio-app-rdm/issues/256

Current code:
![Screenshot 2020-11-06 at 17 03 56](https://user-images.githubusercontent.com/6756943/98388725-6a233a00-2053-11eb-8aa7-2bf22ebe08a9.png)

Example with according to issue:
![Screenshot 2020-11-06 at 17 19 38](https://user-images.githubusercontent.com/6756943/98389460-617f3380-2054-11eb-9ba2-a551cd1ecc40.png)

More zoom to display the shadow:
![Screenshot 2020-11-06 at 17 20 00](https://user-images.githubusercontent.com/6756943/98389478-67751480-2054-11eb-88f0-af9b6676e0ff.png)

lnielsen commented:
Looks good! Here my comments:

- Colors:
   - Use blue from the header (left side) as the background color of the top footer
   - Darken that blue (see Zenodo SCSS for a good value to darken it with)
- Shadows looks good
- Would put the language in the bottom footer bar, but smaller size.
- Menu:
   - Less spacing between menu items (note the menu header is fine.
   - Colors of the menu items you could do with specifying white and make the color a bit transparent.

zzacharo commented on invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/footer.less:
```diff
@@ -6,26 +6,31 @@
-   color: #fff;
-   margin-top: auto;
+.footer-global {
+  color: #ffffff;
```
maybe define also here and a variable e.g `footerTextColor` and set it in the `variables.less`? Similar to what you've done for the `footerDarkColor`...

ppanero commented:
Applied changes:

- Made footer text colors into variables
- Changed footer colors, made light equal than the top and used Zenodo's dark blue
- Moved language to lower footer
- Reduced spacing between menu items
- Made link color white with transparency

![Screenshot 2020-11-09 at 09 39 26](https://user-images.githubusercontent.com/6756943/98518495-bfd42e00-226f-11eb-9a15-8cc1dae1c857.png)
